packages/contracts/src/core/dao/DAO.sol

```diff
@@ -60,15 +60,24 @@ contract DAO is
     /// @notice The internal constant storing the maximal action array length.
     uint256 internal constant MAX_ACTIONS = 256;
 
+    uint256 private constant _NOT_ENTERED = 1;
+    uint256 private constant _ENTERED = 2;
```

Comment:
Wouldn't it be simpler to have a zero value by default (NOT_ENTERED) which doesn't need initialization in the constructor?

Reply:
0 has a different cost associated in the evm. setting it to 1 and 2 gives us a gas refund and thus the tx only pays one SSTORE compared to 2